### PR TITLE
Implements log print

### PR DESCRIPTION
**Pull request by rogerio-gmm**
Implements log print like it was specified here:
https://github.com/AlissonLinhares/nesemu/blob/master/log/brk.log

Note that instructions that read memory will also need to print
` | MEM[address] = value`

### CODE REVIEW

**Comment by gabrielpreviato** [comment]
Hey @rogerio-gmeirelles thanks for the PR. What are your thoughts in moving the prints statements to another method? Something like `def log(self, memory=None)`. We can call this method after the decode_instruction (or before, I don't record the correct order).

The only change that will be needed in the decode_instructions is returning if the memory was accessed or not and it's value.

**Review by gabrielpreviato — CHANGES_REQUESTED** [review]
I think we should move the print statements to another method.

File: Processor.py [hunk 1]
```diff
@@ -24,28 +22,29 @@ def __init__(self, filename):
...
+            # Log instruction - ONLY FOR TESTING PURPOSES
+            # print('| instruction = ' + hex(instruction), end = ' ')
+
+            # Log registers
+            print('| pc = ' + hex(self.PC.value), end = ' ')
+            print('| a = ' + hex(self.A.value), end = ' ')
+            print('| x = ' + hex(self.X.value), end = ' ')
+            print('| y = ' + hex(self.Y.value), end = ' ')
+            print('| sp = ' + hex(self.STACK.value), end = ' ')
+            print('| p[NV-BDIZC] = ' + str(self.FLAGS.get_N()) + str(self.FLAGS.get_V()) + '1' + str(self.FLAGS.get_B()) + str(self.FLAGS.get_D()) + str(self.FLAGS.get_I()) + str(self.FLAGS.get_Z()) + str(self.FLAGS.get_C()), end = '')
```
Comment (victorhenriquetx) [inline Processor.py:36]: Testei aqui e o método get_"FLAG"() printa o valor inteiro da flag, na minha branch eu criei os métodos is_"FLAG"(self) que retorna o bit da flag

**Review by gabrielpreviato — DISMISSED** [review]
Due to not having much time right now, I think we can keep with this implementation. We can make improvements later.

File: Processor.py (hunk 1 above)
Comment (gabrielpreviato) [reply Processor.py:36]: Actually I think this was a problem with this implementation, at least I think @rogerio-gmeirelles intentions were to return 1 or 0.

**Review by gabrielpreviato — APPROVED** [review]

**Pull request closed without merging**